### PR TITLE
fix narrowing warnings

### PR DESCRIPTION
**Pull request by DanielaE**
Signed-off-by: Daniela Engert <dani@ngrt.de>

### CODE REVIEW

File: include/boost/beast/websocket/detail/mask.hpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ mask_inplace(boost::asio::mutable_buffer& b, prepared_key& key)
     {
         for(std::size_t i = 0; i < n; ++i)
             p[i] ^= mask[i];
-        rol(key, 4 - n);
```
Comment (vinniefalco) [inline include/boost/beast/websocket/detail/mask.hpp]: I guess this is causing warnings for 64-bit builds?

Comment (cstratopoulos) [reply include/boost/beast/websocket/detail/mask.hpp]: @vinniefalco That's been my experience with x64 builds on Visual Studio with recent Beast

Comment (DanielaE) [reply include/boost/beast/websocket/detail/mask.hpp]: @vinniefalco : correct. I do all Boost testing on 64-bit msvc at /W4 to catch such (and similar) kinds of problems.

**Comment by vinniefalco** [comment]
@cstratopoulos Yes, I think that is a good idea

**Comment by vinniefalco** [comment]
Merged in 163, thank you!

**Comment by DanielaE** [comment]
Thanks, Vinnie - the narrowing warning is gone now 😄

**Pull request closed without merging**